### PR TITLE
[16.0][MIG] auth_oauth_multi_token

### PR DESCRIPTION
**Pull request by mlaitinen**
- Applied English capitalization rules in field names
- Improved the `res.users` form by horizontally "unsqueezing" the list of auth tokens

### CODE REVIEW

**Comment by mlaitinen** [comment]
I didn't touch `README.rst` because IIRC it gets automatically generated once the PR is merged. Correct me if I'm wrong.

File: auth_oauth_multi_token/models/auth_oauth_multi_token.py [hunk 1]
```diff
@@ -10,7 +10,7 @@ class AuthOauthMultiToken(models.Model):
     """Define a set of tokens."""
 
     _name = "auth.oauth.multi.token"
-    _description = "OAuth2 token"
+    _description = "OAuth2 Token"
```
Comment (pedrobaeza) [inline auth_oauth_multi_token/models/auth_oauth_multi_token.py:13]: Don't change existing capitalization. It's not incorrect as it was. Same comment for the rest of this kind of changes.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
The README is regenerated on merge, yes.

File: auth_oauth_multi_token/models/auth_oauth_multi_token.py (hunk 1 above)
Comment (mlaitinen) [reply auth_oauth_multi_token/models/auth_oauth_multi_token.py:13]: Is this an OCA policy? If yes, could you please send me a link? Only remark about capitalization I could find is here: [CONTRIBUTING.rst](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#fields).

This is the way how titles in English are capitalized, and also how the names of all models & fields in Odoo are formatted.

**Pull request closed without merging**